MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/dds/BathymetryActivity.java

```diff
@@ -62,32 +63,37 @@ protected void onCreate(Bundle savedInstanceState) {
   }
 
   @Override
-  public void onMapReady(MapboxMap mapboxMap) {
-    BathymetryActivity.this.mapboxMap = mapboxMap;
-
-    // Set bounds to Lawrence Lake, Michigan
-    mapboxMap.setLatLngBoundsForCameraTarget(LAKE_BOUNDS);
-
-    // Remove lake label layer
-    mapboxMap.removeLayer("water-label");
-
-    // Initialize FeatureCollection object for future use with layers
-    featureCollection = FeatureCollection.fromJson(loadGeoJsonFromAsset("bathymetry-data.geojson"));
-
-    // Retrieve GeoJSON from local file and add it to the map
-    GeoJsonSource geoJsonSource = new GeoJsonSource(geojsonSourceId,
-      featureCollection);
-    mapboxMap.addSource(geoJsonSource);
-
-    setUpDepthFillLayers();
-    setUpDepthNumberSymbolLayer();
+  public void onMapReady(@NonNull final MapboxMap mapboxMap) {
+
+    mapboxMap.setStyle(Style.OUTDOORS, new Style.OnStyleLoaded() {
+      @Override
+      public void onStyleLoaded(@NonNull Style style) {
+        // Set bounds to Lawrence Lake, Michigan
+        mapboxMap.setLatLngBoundsForCameraTarget(LAKE_BOUNDS);
+
+        // Remove lake label layer
+        style.removeLayer("water-label");
+
+        // Initialize FeatureCollection object for future use with layers
+        featureCollection = FeatureCollection.fromJson(loadGeoJsonFromAsset(
+            "bathymetry-data.geojson"));
+
+        // Retrieve GeoJSON from local file and add it to the map
+        GeoJsonSource geoJsonSource = new GeoJsonSource(GEOJSON_SOURCE_ID,
+          featureCollection);
+        style.addSource(geoJsonSource);
```

Comment:
inline this to `style.addSource(new GeoJsonSource())`?

Reply:
fixed